data/transit/route/bus.json

```diff
@@ -1270,11 +1270,11 @@
     },
     {
       "displayName": "BJBUS",
-      "id": "bjbus-a45453",
+      "id": "bjbus-a1f154",
       "locationSet": {"include": ["cn"]},
       "tags": {
         "network": "BJBUS",
-        "network:wikidata": "Q4881181",
+        "network": "Q4881181",
```

Comment:
Hello, my friend, why delete the ```:wikidata``` ? There will be 2 same ```network``` key but have different values

Reply:
Yes I think this must have been an accident - when I ran the `npm run build` it caught the error, so I fixed it in 03bbcf60a